core/chains/evm/txmgr/common.go

```diff
@@ -24,7 +22,7 @@ const batchSendTransactionTimeout = 30 * time.Second
 // elements in that batch.
 func batchSendTransactions(
 	ctx context.Context,
-	db *sqlx.DB,
+	orm *ORM,
```

Comment:
i was forced to refactor this method in this PR specifically because resender calls this method and no longer has access to the DB object